res/md/common-intro.md

```diff
@@ -1,14 +1,14 @@
 # Preface
 ## I. Aim of the Specification
-This document is one of several related specifications which aim to provide a common set of usage descriptions of international standards for packaging digital information for archiving purposes. These specifications are based on common, international standards for transmitting, describing and preserving digital data. They also utilise the Reference Model for an Open Archival Information System (OAIS) which has Information Packages as its foundation. Familiarity with the core functional entities of OAIS is a prerequisite for understanding the specifications.
+This document is one of several related specifications which aim to provide a common set of usage descriptions of international standards for packaging digital information for archiving purposes. These specifications are based on common, international standards for transmitting, describing and preserving digital data. They also utilise the Reference Model for an Open Archival Information System (OAIS), which has Information Packages as its foundation. Familiarity with the core functional entities of OAIS is a prerequisite for understanding the specifications.
 
-The specifications are designed to help data creators, software developers and digital archives to tackle the challenge of short-, medium- and long-term data management and reuse in a sustainable, authentic, cost-efficient, manageable and interoperable way.
+The specifications are designed to help data creators, software developers, and digital archives to tackle the challenge of short-, medium- and long-term data management and reuse in a sustainable, authentic, cost-efficient, manageable and interoperable way.
 A visualisation of the current specification network can be seen here:
 
 <a name="figi-dip"></a>
 ![OAIS Entities](figs/fig_1_dip.svg "Diagram showing E-ARK specification dependency hierarchy")
 
-**Figure I:** Diagram showing E-ARK specification dependency hierarchy.
+**Figure I:** Diagram showing E-ARK specification dependency hierarchy. Note that the image only shows examples of the available CITS.
```

Comment:
Note that the image only shows a selection of the published CITS and isn't an exhaustive list.

Reply:
Updated!